### PR TITLE
First milestone: Mobile skeleton

### PR DESCRIPTION
**Pull request by AkashaRojee**
Hello!

I set up the portfolio and built and styled the mobile skeleton. Please, review the code.

The mobile skeleton contains the following:

- Navigation menu
- Headline section

I used Flexbox for the positioning. I also fixed linter errors.

Please, click [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to view the Figma design that was used.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Feel free to check the [issues page](https://github.com/AkashaRojee/portfolio/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (arslanbisharat) [inline README.md:71]: - [x] There is no MIT.md file present in your project directory. Please add a file so this link works.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <title>Portfolio</title>
+  </head>
+  <body>
+
+    <!-- Navigation bar -->
+    <nav class="flex-row justify-between align-center main-purple">
+        <a href="https://akasharojee.github.io/portfolio/"><img src="images/page-logo.png" alt="Page logo"></a>
+        <button type="button"><img src="images/menu-icon.png" alt="Menu icon"></button>
+    </nav>
+
```
Comment (arslanbisharat) [inline index.html]: - [x] Navigation Bar section should be enclosed into header-tag.

File: css/style.scss [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+header {
+  padding: 114px 24px;
+  background-image: url('../images/header-background.png');
+  background-repeat: no-repeat;
+
+  >:nth-child(-n+3) {
+    padding-bottom: 12px;
+  }
+
+  #social-icons {
```
Comment (arslanbisharat) [inline css/style.scss]: - [x] Please avoid using id as a selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+          </a>
+        </div>
+        <div class="flex-row align-center">
+          <a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+            <img src="images/social-icons/angellist.png" alt="AngelList icon">
+          </a>
+        </div>
+      </div>
+      
+    </header>
```
Comment (arslanbisharat) [inline index.html]: - [x] This should be using section tag instead of using header tag.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+          <a href="https://github.com/AkashaRojee" target="_blank" rel="noopener noreferrer">
+            <img src="images/social-icons/github.png" alt="GitHub icon">
+          </a>
+        </div>
+        <div class="flex-row align-center">
+          <a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+            <img src="images/social-icons/angellist.png" alt="AngelList icon">
+          </a>
+        </div>
+      </div>
```
Comment (arslanbisharat) [inline index.html]: - [x] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+  padding: 114px 24px;
+  background-image: url("../images/header-background.png");
+  background-repeat: no-repeat;
+}
+
+header > :nth-child(-n+3) {
+  padding-bottom: 12px;
+}
+
+header #social-icons {
```
Comment (arslanbisharat) [inline css/style.css]: Please avoid using id as a selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+#### 1. Build mobile skeleton
+
+* Parse a Figma design
+* Use Flexbox for positioning
+* Use images and backgrounds
+
+## Built With
+
+- Major languages: HTML, CSS
+- Frameworks: none
```
Comment (arslanbisharat) [inline README.md]: [ ] [Optional] You can remove` Frameworks: none ` line to avoid confusion.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @AkashaRojee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (AkashaRojee) [reply README.md:71]: [File](https://github.com/AkashaRojee/portfolio/blob/mobile-skeleton/MIT.md) is now added.

File: index.html (hunk 2 above)
Comment (AkashaRojee) [reply index.html]: nav element has now been placed in header element.

File: css/style.scss (hunk 3 above)
Comment (AkashaRojee) [reply css/style.scss]: Social icons section has now been reworked accordingly.

Comment (AkashaRojee) [reply css/style.scss]: The ID "social-icons" has now been removed and the class "social" is used instead.

File: index.html (hunk 5 above)
Comment (AkashaRojee) [reply index.html]: Social icons section has now been reworked accordingly.

File: index.html (hunk 4 above)
Comment (AkashaRojee) [reply index.html]: header element here has now been replaced with section element.

File: css/style.css (hunk 6 above)
Comment (AkashaRojee) [reply css/style.css]: The ID "social-icons" has now been removed and the class "social" is used instead.

File: README.md (hunk 7 above)
Comment (AkashaRojee) [reply README.md]: Frameworks text has been removed.

**Comment by AkashaRojee** [comment]
Hi @arslanbisharat,

Thanks for your feedback. Here is a summary of the implemented changes as suggested:

- Add MIT.md file
- Place nav in header
- Replace ID social-icons with class social
- Place headline section in a section element instead of a header element
- Rework social icons section using list
- Remove Frameworks text from README

Thanks!

File: README.md (hunk 1 above)
Comment (descholar-ceo) [reply README.md:71]: :+1: This issue has been addressed successfully :heavy_check_mark:

File: index.html (hunk 2 above)
Comment (descholar-ceo) [reply index.html]: :+1: This issue has been addressed successfully :heavy_check_mark:

File: css/style.scss (hunk 3 above)
Comment (descholar-ceo) [reply css/style.scss]: :+1: This issue has been addressed successfully :heavy_check_mark: 

=> But look at the `#headline` id as well, that should be addressed just like this one

File: index.html (hunk 4 above)
Comment (descholar-ceo) [reply index.html]: :+1: This issue has been addressed successfully :heavy_check_mark:

File: index.html (hunk 5 above)
Comment (descholar-ceo) [reply index.html]: :+1: This issue has been addressed successfully :heavy_check_mark:

File: css/style.css (hunk 6 above)
Comment (descholar-ceo) [reply css/style.css]: :+1: This issue has been addressed successfully :heavy_check_mark: 

=> But as I mentioned earlier, you can checkout the `#headline` id as well and address it

**Review by descholar-ceo — APPROVED** [review]
Hi @AkashaRojee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AkashaRojee** [comment]
Hi @descholar-ceo,

Thanks!

**Pull request merged**